drivers/gpu/drm/bridge/cdns-mhdp.c

```diff
@@ -571,6 +571,12 @@ static unsigned int max_link_rate(struct cdns_mhdp_host host,
 	return min(host.link_rate, sink.link_rate);
 }
 
+static u8 max_num_lanes(struct cdns_mhdp_host host,
+			struct cdns_mhdp_sink sink)
+{
+	return min_t(u8, sink.lanes_cnt, host.lanes_cnt & GENMASK(2, 0));
```

Comment:
Why do we need to mask the host lanes_cnt? I see it in the old code too, but it's not clear why we do that.